app/models/pawn.rb

```diff
@@ -1,2 +1,18 @@
 class Pawn < Piece
+
+  def valid_move?(end_x, end_y)
+    y_diff = end_y - position_y if white?
+    y_diff = position_y - end_y if black?
+    
+    forward_move?(end_y, y_diff) ? true : false
```

Comment:
i think we can use if piece.present? combined with these end_y definitions you've written to make sure the pawn doesn't move into another piece's square, but that can wait for a refactor.